app-modules/assist-data-model/src/Filament/Resources/StudentResource/Pages/ManageStudentAlerts.php

```diff
@@ -78,9 +77,16 @@ public function form(Form $form): Form
                     ->required(),
                 Select::make('severity')
                     ->options(AlertSeverity::class)
-                    ->enum(AlertSeverity::class),
```

Comment:
Let's bring this `enum` call back here for the validation that an ENUM is selected here.